### PR TITLE
Fix README TypeError when one container is running

### LINKED ISSUE #379: Example from Readme fails with TypeError
For me running the example from the Readme file results in:

```
Traceback (most recent call last):
[...]
    first_name = container['Names'][0]
TypeError: 'Container' object is not subscriptable
```

It seems that `Container` does not implement `__getitem__(self, key)`. However, I think probably `container.name` was meant to be used.

I think it would also be good to clarify why

```
container = client.containers.get(first_name)
```

is used in the example, as it seems to return the same container as already retrieved from the list. Perhaps, the line could be omitted or a clarifying comment or an `assert` showing that both objects are equal could be added.

Edit: for the problem to show a container must be running, since otherwise the loop will not be entered.

Python: 3.10.12
podman==4.9.0 (podman-py)
OS: Ubuntu 22.04

### PR DESCRIPTION
**Pull request by inknos**
closes #379

PTAL @jwhonce

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,9 +35,11 @@ with PodmanClient(base_url=uri) as client:
 
     # find all containers
     for container in client.containers.list():
-        first_name = container['Names'][0]
-        container = client.containers.get(first_name)
+        # it is important to get the container object again with the get()
+        # method to get all the fields like container.status populated
+        container = client.containers.get(container.name)
```
Comment (jwhonce) [inline README.md:40]:
```suggestion
        container.reload()
```
would also populate all the fields.

PS. The comment could also include a notice that sparse=True is our default. Future work could allow sparse=False on the list() call.

Comment (inknos) [reply README.md]: thanks @jwhonce, good idea. I committed new changes

**Comment by jwhonce** [comment]
LGTM

**Pull request merged**